### PR TITLE
Better pytest integration

### PR DESCRIPTION
**Pull request by sschneid**
- `--pytest-option` (`-p`) flag to pass options on to pytest
- single-character options flags
- improved `--help` output
- bump to `1.1.2`

### CODE REVIEW

File: shakedown/cli/main.py [hunk 1]
```diff
@@ -9,20 +9,20 @@
 
 @click.command('shakedown')
 @click.argument('path', type=click.Path(exists=True), nargs=-1)
-@click.option('--dcos-url', help='URL to a running DCOS cluster.')
-@click.option('--fail', type=click.Choice(['fast', 'never']), help='Sepcify whether to continue testing when encountering failures. (default: fast)')
-@click.option('--ssh-key-file', type=click.Path(), help='Path to the SSH keyfile to use for authentication')
+@click.option('-u', '--dcos-url', help='URL to a running DCOS cluster.')
+@click.option('-f', '--fail', type=click.Choice(['fast', 'never']), help='Sepcify whether to continue testing when encountering failures. (default: fast)')
+@click.option('-k', '--ssh-key-file', type=click.Path(), help='Path to the SSH keyfile to use for authentication.')
```
Comment (colin-msphere) [inline shakedown/cli/main.py]: `ssh` uses `-i` for the key file.

File: shakedown/cli/main.py [hunk 2]
```diff
@@ -9,20 +9,20 @@
...
+@click.option('-q', '--quiet', is_flag=True, help='Suppress all superfluous output.')
+@click.option('-n', '--ssl-no-verify', is_flag=True, help='Suppress SSL certificate verification.')
```
Comment (colin-msphere) [inline shakedown/cli/main.py]: `curl` uses `-k` for ignoring certs.

File: shakedown/cli/main.py [hunk 3]
```diff
@@ -9,20 +9,20 @@
...
+@click.option('-o', '--stdout', type=click.Choice(['pass', 'fail', 'skip', 'all', 'none']), help='Print the standard output of tests with the specified result. (default: fail)')
+@click.option('-i', '--stdout-inline', is_flag=True, help='Display output inline rather than after test phase completion.')
+@click.option('-p', '--pytest-option', multiple=True, help='Options flags to pass to pytest.')
```
Comment (colin-msphere) [inline shakedown/cli/main.py]: 👍

File: shakedown/cli/main.py (hunk 2 above)
Comment (sschneid) [inline shakedown/cli/main.py]: Updated.

**Pull request merged**